lib/compat/wordpress-6.0/block-editor-settings.php

```diff
@@ -33,6 +33,21 @@ function_exists( 'gutenberg_is_edit_site_page' ) &&
 		$context = 'mobile';
 	}
 
+	if ( 'site-editor' === $context ) {
+		// Remove global styles added by core.
+		// This needs to be fixed in core but this will help us in the meanwhile.
+		$styles_without_existing_global_styles = array();
+		foreach ( $settings['styles'] as $style ) {
+			if (
+				! isset( $style['__unstableType'] ) ||
+				! in_array( $style['__unstableType'], array( 'globalStyles', 'presets' ), true )
```

Comment:
If I'm not wrong that's not enough. Even the "theme" one is a global styles one. I'm using "emptytheme" and it doesn't have any custom CSS file. So I expect the style array to be empty?

Reply:
How can we differentiate between the styles that are generated ant the ones that come from the files.